ddtrace/llmobs/_llmobs.py

```diff
@@ -785,6 +787,127 @@ def _tag_metrics(span: Span, metrics: Dict[str, Any]) -> None:
             return
         span._set_ctx_item(METRICS, metrics)
 
+    @classmethod
+    def submit_evaluation_for(
+        cls,
+        label: str,
+        metric_type: str,
+        value: Union[str, int, float],
+        span: Optional[dict] = None,
+        span_with_tag_value: Optional[Dict[str, str]] = None,
+        tags: Optional[Dict[str, str]] = None,
+        ml_app: Optional[str] = None,
+        timestamp_ms: Optional[int] = None,
+    ) -> None:
+        """
+        Submits a custom evaluation metric for a given span.
+
+        :param str label: The name of the evaluation metric.
+        :param str metric_type: The type of the evaluation metric. One of "categorical", "score".
+        :param value: The value of the evaluation metric.
+                      Must be a string (categorical), integer (score), or float (score).
+        :param dict span: A dictionary of shape {'span_id': str, 'trace_id': str} uniquely identifying
+                            the span associated with this evaluation.
+        :param tuple span_with_tag_value: A dictionary of shape {'tag_key': str, 'tag_value': str} uniquely identifying
+                            the span associated with this evaluation.
+        :param tags: A dictionary of string key-value pairs to tag the evaluation metric with.
+        :param str ml_app: The name of the ML application
+        :param int timestamp_ms: The timestamp in milliseconds when the evaluation metric result was generated.
```

Comment:
```suggestion
        :param int timestamp_ms: The unix timestamp in milliseconds when the evaluation metric result was generated.
```
Maybe worth specifying what type of timestamp to be exact

Reply:
Also should mention that if no timestamp is provided we will default to the current time